internal/provider/camunda_channel_data_source.go

```diff
@@ -173,6 +173,6 @@ func (d *CamundaChannelDataSource) Read(ctx context.Context, req datasource.Read
 
 	resp.Diagnostics.AddError(
 		"Client Error",
-		fmt.Sprintf("Camunda Cloud channel '%s' not founds", data.Name.ValueString()),
+		fmt.Sprintf("Camunda Cloud channel '%s' not found.", data.Name.ValueString()),
```

Comment:
💭 Does Hashicorp/TF a style guide for error formatting? 

In Go it usually is the following. But I think it does not apply to these type of errors since they are usually not wrapped 🤔 . 

> Lastly, error messages are usually written in lower-case and don’t end in punctuation. Exceptions can be made though, for example when including a proper noun, a function name that begins with a capital letter, etc.

Reply:
The diagnostics are directly reported to the end user through Terraform "UI" (CLI or web), in this case it makes sense to have properly formatted messages. (Think more about it as a message to be displayed to the user, to report an error in this case)

Reply:
The doc doesn't provide guidelines, but it's also not only returning errors to the user (you can also provide warnings for instance): https://developer.hashicorp.com/terraform/plugin/framework/diagnostics#detail